src/MultiBridgeMessageReceiver.sol

```diff
@@ -20,7 +20,10 @@ import "./libraries/Message.sol";
 /// governance timelock contract.
 /// @dev The contract only accepts messages from trusted bridge receiver adapters, each of which implements the
 /// IMessageReceiverAdapter interface.
```

Comment:
can you run the linter, please? the build is failing

Reply:
Done. VSCode's been having issues.